src/olympia/users/management/commands/createsuperuser.py

```diff
@@ -80,6 +99,29 @@ def handle(self, *args, **options):
                 'api-secret': apikey.secret
             }))
 
+        if options.get('save_api_credentials', False):
+            # json object for variables file
+            variables = {}
+            variables['api'] = {}
+
+            # set hostname to stdin or env variable
+            if options.get('hostname', False):
+                hostname = options.get('hostname')
+            else:
+                hostname = os.environ['PYTEST_BASE_URL']
+
+            # export to variables.json object
+            variables['api'].update({
+                hostname: dict([
+                    ('username', user.username,),
+                    ('jwt_issuer', apikey.key),
+                    ('jwt_secret', apikey.secret)
+                ])
+            })
```

Comment:
Can you please revisit this and cleanup the code a bit based on what I suggested in https://github.com/mozilla/addons-server/pull/4624/files#r99990992 ?